packages/ui/src/theme/utils.ts

```diff
@@ -65,7 +65,7 @@ export function isDesignToken(value: unknown): value is WebDesignToken {
   return isObject(value) && has(value, 'value');
 }
 
-export function isShadowToken(value: unknown): value is ShadowValue {
+export function isShadowToken(value: unknown): value is ShadowValue & object {
```

Comment:
Fairly certain that the definition of `ShadowValue` can resolve to either an `object` of shadow props or `string`. Not fully related to this PR but maybe we should rename this to something more explicit

Reply:
good point, `isShadowObject` it is